2025-01-31 13:11:41,990 INFO: Application startup in development mode [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:84]
2025-01-31 13:11:41,991 INFO: Running with BASE_DIR: c:\Users\BJ.Marshall\Documents\python\REI-Tracker [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:101]
2025-01-31 13:11:41,992 INFO: Environment: Development [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:102]
2025-01-31 13:11:42,290 INFO: All Dash apps initialized successfully [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:176]
2025-01-31 13:11:43,026 INFO: Application startup in development mode [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:84]
2025-01-31 13:11:43,027 INFO: Running with BASE_DIR: c:\Users\BJ.Marshall\Documents\python\REI-Tracker [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:101]
2025-01-31 13:11:43,030 INFO: Environment: Development [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:102]
2025-01-31 13:11:43,321 INFO: All Dash apps initialized successfully [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:176]
2025-01-31 13:24:26,799 INFO: Application startup in development mode [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:84]
2025-01-31 13:24:26,799 INFO: Running with BASE_DIR: c:\Users\BJ.Marshall\Documents\python\REI-Tracker [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:101]
2025-01-31 13:24:26,799 INFO: Environment: Development [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:102]
2025-01-31 13:24:27,094 INFO: All Dash apps initialized successfully [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:176]
2025-02-03 07:17:59,830 INFO: Application startup in development mode [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:84]
2025-02-03 07:17:59,830 INFO: Running with BASE_DIR: c:\Users\BJ.Marshall\Documents\python\REI-Tracker [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:101]
2025-02-03 07:17:59,830 INFO: Environment: Development [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:102]
2025-02-03 07:18:00,760 INFO: All Dash apps initialized successfully [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:176]
2025-02-03 07:18:01,850 INFO: Application startup in development mode [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:84]
2025-02-03 07:18:01,850 INFO: Running with BASE_DIR: c:\Users\BJ.Marshall\Documents\python\REI-Tracker [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:101]
2025-02-03 07:18:01,850 INFO: Environment: Development [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:102]
2025-02-03 07:18:02,183 INFO: All Dash apps initialized successfully [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:176]
2025-02-03 07:39:14,372 INFO: View transactions accessed by user: {'id': 'bjmar867@gmail.com', 'name': 'BJ Marshall', 'role': 'Admin', 'is_authenticated': True} [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\routes\transactions.py:454]
2025-02-03 07:40:53,309 INFO: View transactions accessed by user: {'id': 'bjmar867@gmail.com', 'name': 'BJ Marshall', 'role': 'Admin', 'is_authenticated': True} [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\routes\transactions.py:454]
2025-02-03 10:21:19,876 INFO: Application startup in development mode [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:84]
2025-02-03 10:21:19,876 INFO: Running with BASE_DIR: c:\Users\BJ.Marshall\Documents\python\REI-Tracker [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:101]
2025-02-03 10:21:19,876 INFO: Environment: Development [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:102]
2025-02-03 10:21:20,896 INFO: All Dash apps initialized successfully [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:176]
2025-02-03 10:21:21,729 INFO: Application startup in development mode [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:84]
2025-02-03 10:21:21,729 INFO: Running with BASE_DIR: c:\Users\BJ.Marshall\Documents\python\REI-Tracker [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:101]
2025-02-03 10:21:21,730 INFO: Environment: Development [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:102]
2025-02-03 10:21:22,041 INFO: All Dash apps initialized successfully [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:176]
2025-02-03 10:38:03,649 INFO: View transactions accessed by user: {'id': 'bjmar867@gmail.com', 'name': 'BJ Marshall', 'role': 'Admin', 'is_authenticated': True} [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\routes\transactions.py:454]
2025-02-03 10:38:26,434 INFO: View transactions accessed by user: {'id': 'bjmar867@gmail.com', 'name': 'BJ Marshall', 'role': 'Admin', 'is_authenticated': True} [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\routes\transactions.py:454]
2025-02-03 10:38:44,143 INFO: View transactions accessed by user: {'id': 'bjmar867@gmail.com', 'name': 'BJ Marshall', 'role': 'Admin', 'is_authenticated': True} [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\routes\transactions.py:454]
2025-02-03 10:39:49,717 INFO: View transactions accessed by user: {'id': 'bjmar867@gmail.com', 'name': 'BJ Marshall', 'role': 'Admin', 'is_authenticated': True} [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\routes\transactions.py:454]
2025-02-03 10:45:05,269 INFO: View transactions accessed by user: {'id': 'bjmar867@gmail.com', 'name': 'BJ Marshall', 'role': 'Admin', 'is_authenticated': True} [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\routes\transactions.py:454]
2025-02-03 10:45:09,800 ERROR: File not found: c:\Users\BJ.Marshall\Documents\python\REI-Tracker\data\uploads\reimb_2025.01_Brian_Reimbursements.png [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\routes\transactions.py:480]
2025-02-03 10:45:09,802 ERROR: Error serving artifact reimb_2025.01_Brian_Reimbursements.png: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again. [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\routes\transactions.py:490]
2025-02-03 10:45:09,829 ERROR: Full error traceback: Traceback (most recent call last):
  File "c:\Users\BJ.Marshall\Documents\python\REI-Tracker\routes\transactions.py", line 481, in get_artifact
    abort(404)
  File "C:\Users\BJ.Marshall\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\flask\helpers.py", line 272, in abort
    current_app.aborter(code, *args, **kwargs)
  File "C:\Users\BJ.Marshall\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\exceptions.py", line 863, in __call__
    raise self.mapping[code](*args, **kwargs)
werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
 [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\routes\transactions.py:491]
2025-02-03 10:45:20,769 INFO: View transactions accessed by user: {'id': 'bjmar867@gmail.com', 'name': 'BJ Marshall', 'role': 'Admin', 'is_authenticated': True} [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\routes\transactions.py:454]
2025-02-03 10:46:50,409 INFO: View transactions accessed by user: {'id': 'bjmar867@gmail.com', 'name': 'BJ Marshall', 'role': 'Admin', 'is_authenticated': True} [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\routes\transactions.py:454]
2025-02-03 10:52:33,284 INFO: View transactions accessed by user: {'id': 'bjmar867@gmail.com', 'name': 'BJ Marshall', 'role': 'Admin', 'is_authenticated': True} [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\routes\transactions.py:454]
2025-02-03 10:52:47,911 INFO: View transactions accessed by user: {'id': 'bjmar867@gmail.com', 'name': 'BJ Marshall', 'role': 'Admin', 'is_authenticated': True} [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\routes\transactions.py:454]
2025-02-03 10:53:03,740 INFO: View transactions accessed by user: {'id': 'bjmar867@gmail.com', 'name': 'BJ Marshall', 'role': 'Admin', 'is_authenticated': True} [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\routes\transactions.py:454]
2025-02-04 08:13:57,374 INFO: Application startup in development mode [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:84]
2025-02-04 08:13:57,375 INFO: Running with BASE_DIR: c:\Users\BJ.Marshall\Documents\python\REI-Tracker [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:101]
2025-02-04 08:13:57,376 INFO: Environment: Development [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:102]
2025-02-04 08:13:58,275 INFO: All Dash apps initialized successfully [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:176]
2025-02-04 08:13:59,073 INFO: Application startup in development mode [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:84]
2025-02-04 08:13:59,074 INFO: Running with BASE_DIR: c:\Users\BJ.Marshall\Documents\python\REI-Tracker [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:101]
2025-02-04 08:13:59,075 INFO: Environment: Development [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:102]
2025-02-04 08:13:59,352 INFO: All Dash apps initialized successfully [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:176]
2025-02-04 08:16:36,779 INFO: View transactions accessed by user: {'id': 'bjmar867@gmail.com', 'name': 'BJ Marshall', 'role': 'Admin', 'is_authenticated': True} [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\routes\transactions.py:454]
2025-02-04 08:16:59,521 INFO: Processing POST request to edit transaction ID: 175 [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\routes\transactions.py:328]
2025-02-04 08:20:22,243 INFO: Application startup in development mode [in c:\Users\BJ.Marshall\Documents\python\REI-Tracker\__init__.py:84]
